src/type-extensions.ts

```diff
@@ -165,6 +165,12 @@ declare module "hardhat/types/runtime" {
          * @returns void
          */
         load: (path: string) => Promise<void>;
+
+        /**
+         * Creates an empty block
+         * @returns void
+         */
+        createBlock: () => Promise<void>;
```

Comment:
Should it be void? Can't the already existing Block type (defined in starknet-types.ts) be used here?